debvader/model.py

```diff
@@ -207,12 +215,18 @@ def load_deblender(
         experimental_run_tf_function=False,
     )
 
-    # Load the weights corresponding to the chosen survey
-    data_path = pkg_resources.resource_filename("debvader", "data/")
-    loading_path = os.path.join(data_path, "weights", survey, "not_normalised", "loss")
-    print(loading_path)
-    latest = tf.train.latest_checkpoint(loading_path)
-    net.load_weights(latest)
+    if external_path_to_folder is None:
+        # Load the weights corresponding to the chosen survey
+        data_path = pkg_resources.resource_filename("debvader", "data/")
+        loading_path = os.path.join(
+            data_path, "weights", survey, "not_normalised", "loss"
+        )
+        print(loading_path)
```

Comment:
Maybe add something to the printed line like

"Loading weights from {loading_path}"